apps/condo/domains/billing/components/Settings/BillingChooser.tsx

```diff
@@ -0,0 +1,109 @@
+
+    const OneBillingWarningMessage = intl.formatMessage({ id: 'OneBillingWarning' })
+    const NoPermissionMessage = intl.formatMessage({ id: 'NoPermissionToSettings' })
+
+    const userOrganization = useOrganization()
+    const organizationId = get(userOrganization, ['organization', 'id'])
+    const canManageIntegrations = get(userOrganization, ['link', 'role', 'canManageIntegrations'], false)
+    const {
+        objs: integrations,
+        loading: integrationsLoading,
+        error: integrationsError,
+    } = BillingIntegration.useObjects({})
+
+    const {
+        obj: currentContext,
+        error: contextError,
+        loading: contextLoading,
+    } = BillingIntegrationOrganizationContext.useObject({
+        where: {
+            organization: {
+                id: organizationId,
+            },
+        },
+    }, {
+        fetchPolicy: 'network-only',
+    })
+
+    if (!canManageIntegrations) {
+        return (
+            <BasicEmptyListView>
+                <Typography.Title level={3}>
+                    {NoPermissionMessage}
+                </Typography.Title>
+            </BasicEmptyListView>
+        )
+    }
+
+    if (integrationsLoading || contextLoading) {
+        return (
+            <Loader fill size={'large'}/>
+        )
+    }
+
+    if (integrationsError || contextError) {
+        return (
+            <BasicEmptyListView>
+                <Typography.Title level={4}>
+                    {integrationsError ? integrationsError : contextError}
+                </Typography.Title>
+            </BasicEmptyListView>
+        )
+    }
+
+    return (
+        <>
+            <Space direction={'vertical'} size={40} style={{ width: '100%' }}>
+                {
+                    !currentContext && (
+                        <Alert message={OneBillingWarningMessage} showIcon type="warning" style={{ width: 'fit-content' }} />
+                    )
+                }
+                <Col span={24}>
+                    <Row gutter={[44, 44]}>
+                        {
+                            integrations.map((integration) => {
+                                const isActiveIntegration = !!currentContext && integration.id === currentContext.integration.id
+                                const status = isActiveIntegration
+                                    ? (
+                                        currentContext.status as CardStatuses
+                                    )
+                                    : (
+                                        currentContext ? 'disabled' : 'available'
+                                    )
+                                return (
+                                    <Col
+                                        span={12}
+                                        key={integration.id}
+                                        style={{
+                                            order: isActiveIntegration ? -1 : 'unset',
```

Comment:
What is this?

Reply:
Antd cols are just flex items, so...if I want to move some card on first place, I can just change it's css "order" to negative value

Reply:
Sound like we will have only like <10 integrations available for now. Decided to left this front-end method for now...Since if we'll decide to make it with pagination, we will need custom sorter in useObjects, which is not trivially (at least for now)